test/cases/fixes/incompleteTypes/returnTypes/expected.ts

```diff
@@ -113,6 +116,7 @@
 	};
 
 	const returnsBigintOrNumber = (): number | bigint => {
+// @ts-expect-error -- TODO: BigInt literals are not available when targeting lower than ES2020.
```

Comment:
[Testing] Let's switch the test to use ESNext. Testing for this error isn't the point of the test.